cypress/support/commands/explorer.js

```diff
@@ -105,16 +105,16 @@ Cypress.Commands.add('selectAccordionItem', (accordionPath) => {
           return;
         }
       }
-      // If we reach here, it means the label was not found
+      // Reaching this point indicates the label was not found - throw an error and exit.
+      // Traversing up through the ancestors to get the name of the accordion where the lookup was performed
+      const accordionPanel = expandedAccordion
+        .closest('.panel')
+        .find('.panel-heading h4.panel-title a')
+        .text();
       const errorMessage = `${
         isClickableNode ? 'Target' : 'Intermediate'
-      } node - "${accordionLabel}" was not found`;
-      Cypress.log({
-        name: 'error',
-        displayName: '❗ CypressError:',
-        message: errorMessage,
-      });
-      throw new Error(errorMessage);
+      } node - "${accordionLabel}" was not found in the expanded "${accordionPanel}" accordion panel.`;
```

Comment:
I've not seen what it looks like before and after but we can iterate on it and see what works best for debugging errors.